docs/installation/ca/Installing_CA_with_Secure_Database_Connection.md

```diff
@@ -6,62 +6,32 @@ Overview
 
 This page describes the process to install a CA subsystem with a secure database connection.
 
-Ensure that the secure connection has been enabled on the directory server.
-Export the signing certificate for the directory server into ds_signing.crt.
-This step is described [here](https://www.dogtagpki.org/wiki/DS_SSL).
+Before beginning with the installation, please ensure that you have configured the directory
+server and added base entries.
+The step is described [here](https://github.com/dogtagpki/pki/wiki/DS-Installation).
+
+Export the DS signing certificate for the directory server into `ds_signing.crt`.
+See [Exporting DS Certificates](https://github.com/dogtagpki/pki/wiki/Exporting-DS-Certificates).
 
 CA Subsystem Installation
 -------------------------
 
-Prepare a file (e.g. ca.cfg) that contains the deployment configuration, for example:
-
-```
-[DEFAULT]
-pki_server_database_password=Secret.123
-
-[CA]
-pki_admin_email=caadmin@example.com
-pki_admin_name=caadmin
-pki_admin_nickname=caadmin
-pki_admin_password=Secret.123
-pki_admin_uid=caadmin
-
-pki_client_database_password=Secret.123
-pki_client_database_purge=False
-pki_client_pkcs12_password=Secret.123
-
-pki_ds_ldaps_port=636
-pki_ds_secure_connection=True
-pki_ds_secure_connection_ca_nickname=ds_signing
-pki_ds_secure_connection_ca_pem_file=ds_signing.crt
+Prepare a deployment configuration (e.g. `ca-secure-ds.cfg`) to deploy CA subsystem.
+By default the subsystem will be deployed into a Tomcat instance called `pki-tomcat`.
 
-pki_ds_base_dn=dc=ca,dc=pki,dc=example,dc=com
-pki_ds_database=ca
-pki_ds_password=Secret.123
+A sample deployment configuration is available at [/usr/share/pki/server/examples/installation/ca-secure-ds.cfg](../../../base/server/examples/installation/ca-secure-ds.cfg).
 
-pki_security_domain_name=EXAMPLE
+To start the installation execute the following command:
 
-pki_ca_signing_nickname=ca_signing
-pki_ocsp_signing_nickname=ca_ocsp_signing
-pki_audit_signing_nickname=ca_audit_signing
-pki_sslserver_nickname=sslserver
-pki_subsystem_nickname=subsystem
 ```
-
-Then execute the following command:
-
-```
-$ pkispawn -f ca.cfg -s CA
+$ pkispawn -f ca-secure-ds.cfg -s CA
 ```
 
-It will install CA subsystem in a Tomcat instance (default is pki-tomcat) and create the following NSS databases:
-* server NSS database: /etc/pki/pki-tomcat/alias
-* admin NSS database: ~/.dogtag/pki-tomcat/ca/alias
+CA System Certificates
+----------------------
 
-Verifying System Certificates
------------------------------
-
-Verify that the server NSS database contains the following certificates:
+After installation the DS signing certificate and the CA system certificates with their keys will be stored
+in the server NSS database (i.e. `/etc/pki/pki-tomcat/alias`):
```

Comment:
I'd probably rephrase it slightly to add clarification:
After installation the CA system certificates with their keys will be generated and stored in the server NSS database (i.e. /etc/pki/pki-tomcat/alias), and the DS signing certificate will be added to the same crypto token (is this doc specifically written for soft token only?  If it's clear that is the case, then instead of "crypto token" as I suggested, you could just say "NSS database" ) :